examples/documentation_examples/examples.go

```diff
@@ -2956,3 +2957,172 @@ func StableAPIExamples() {
 	StableAPINonStrictExample()
 	StableAPIDeprecationErrorsExample()
 }
+
+func insertSnapshotQueryTestData(mt *mtest.T) {
+	catColl := mt.CreateCollection(mtest.Collection{Name: "cats"}, true)
+	_, err := catColl.InsertMany(context.Background(), []interface{}{
+		bson.D{
+			{"adoptable", false},
+			{"name", "Miyagi"},
+			{"color", "grey-white"},
+			{"age", 14},
+		},
+		bson.D{
+			{"adoptable", true},
+			{"name", "Joyce"},
+			{"color", "black"},
+			{"age", 10},
+		},
+	})
+	require.NoError(mt.T, err)
```

Comment:
Optional: I believe `mt` satisfies the interface used by all `require` functions, so you should be able to just pass that in.

E.g.
```go
require.NoError(mt, err)
```

That applies for all uses of `require` functions in the PR.